adoc/deployment-sysreqs.adoc

```diff
@@ -355,6 +355,43 @@ net.ipv4.ip_forward = 1
 
 `net.ipv4.ip_forward` must be set to `1`. Additionally, you can check in what order persisted rules are processed by running `sysctl --system -a`.
 
+==== Networking Whitelist
+
+Besides the {suse} provided packages and containers, {productname} is typically used with third party provided containers and charts.
+
+The following {suse} provided resources must be available:
+
+[options="header,autowidth"]
+|===
+|URL|Name|Purpose
+
+|scc.suse.com|{scc}|Allow registration and license activation
+|registry.suse.com|{suse} container registry|Provide container images
+|*.cloudfront.net|Cloudfront|CDN/distribution backend for `registry.suse.com`
+|kubernetes-charts.suse.com|{suse} helm charts repository|Provide helm charts
+|updates.suse.com|{suse} package update channel|Provide package updates
+|===
+
+For additional Upstream / Third-Party resources please also allow the following:
```

Comment:
These are not necessary, even in Airgapped environments, for a clean deployment. Of course, it doesn't mean that much can be done afterwards if the most common helm charts and registries aren't available...

I would rephrase this as:

```
It would also be convenient in your kubernetes usage, to allow network access to following resources:
```

Reply:
I don't understand this view. A "clean" deployment contains no workloads, where would they come from? Certainly not from SUSE because we sell a platform that you can run "whatever" on. If they are in an air gap environment then our main job (besides providing a stable platform and how to set it up) is to explain how to make workloads function. This absolutely requires external charts, images etc.

Reply:
I think we're not yet talking about airgap here, these are just required if you have an internal cluster which is not meant to be air-gapped from Internet but where you still need to filter what is authorized.

Reply:
If you run an airgrap cluster, then you have absolutely no direct access to these third party resources so in that case, it's the job of the cluster administrator to set up internal mirror registries.

But in general, the way I see it for the majority it that, when you run an air-grapped cluster, you don't want to fetch anything from these third party resources and you use only internal stuff.

Reply:
Agreed.